sdk/search/Azure.Search.Documents/src/Indexes/Models/AnalyzeTextOptions.cs

```diff
@@ -45,10 +36,6 @@ public AnalyzeTextOptions(string text, LexicalNormalizerName normalizerName) : t
         [CodeGenMember("Tokenizer")]
         public LexicalTokenizerName? TokenizerName { get; }
 
-        /// <summary> The name of the normalizer to use to normalize the given text. </summary>
-        [CodeGenMember("Normalizer")]
-        public LexicalNormalizerName? NormalizerName { get; }
-
         /// <summary> An optional list of token filters to use when breaking the given text. </summary>
         public IList<TokenFilterName> TokenFilters { get; }
```

Comment:
A few lines below this, since I can't comment on code not in the diff.

`IList<string> CharFilters` is defined in the Swagger as `IList<CharFilter> CharFilters`, do we want to add another property for that? 

Also, this is my lack of understanding .NET code generation and semantics, why is there no setter for this property? This is a request model. Or, is it expected to do something like `AnalyzeTestOptions.CharFilters.add(...)`?

Reply:
>IList<string> CharFilters is defined in the Swagger as IList<CharFilter> CharFilters, do we want to add another property for that?
 
Given that this property has already been released in a GA, I'm uncertain whether we should introduce a similar property, `IList<CharFilterName> CharFilters`, which has only one single enum value. I can, however, enhance the documentation to provide clarity on passing CharFilterName values.

@tg-msft - Do you think we should add `[EditorBrowsable(EditorBrowsableState.Never)]` to the existing property and add  `IList<CharFilterName> CharFilters`?

>Also, this is my lack of understanding .NET code generation and semantics, why is there no setter for this property? This is a request model. Or, is it expected to do something like AnalyzeTestOptions.CharFilters.add(...)?

As per the [.NET guidelines](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections#collection-properties-and-return-values), we don't property setters for collection properties. And Yes, users are expected to utilize `AnalyzeTestOptions.CharFilters.Add(...)`.